magda-web-client/src/index.scss

```diff
@@ -2,7 +2,9 @@
 @import "./pancake/sass/pancake.scss";
 html {
 }
-
+.au-body {
```

Comment:
This still has to be `body` instead of `.au-body` because `.au-body` gets used in a bunch of places including the footer, mobile header, banner :(.